readthedocs/projects/views/private.py

```diff
@@ -100,6 +101,13 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         # Set the default search to search files instead of projects
         context['type'] = 'file'
+
+        filter = ProjectListFilterSet(self.request.GET, queryset=self.get_queryset())
+        context['filter'] = filter
+        context['project_list'] = filter.qs
+        # Alternatively, dynamically override super()-derived `project_list` context_data
+        # context[self.get_context_object_name(filter.qs)] = filter.qs
```

Comment:
Shouldn't this be behind `if settings.RTD_EXT_THEME_ENABLED`?

Reply:
I think you're right. I assumed that I wrote this to replace existing filtering functionality on `project_list`, but digging through code I don't see any similar filter mechanism.